CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -434,6 +439,7 @@ public void LoadContent(ContentManager content)
             _aiTwoIdentifier = new();
             _aiThreeIdentifier = new();
             _aiFourIdentifier = new();
+            _turnIndicator = new(content.Load<Texture2D>("TEMP_CHIP"));
```

Comment:
we should not load content in the constructor of an object. All textures should be loaded in a static texture class, which I believe should already exist for Texas Hold Em. you should then reference that static texture in the draw method for this class.